src/engraving/dom/stem.h

```diff
@@ -67,9 +68,11 @@ class Stem final : public EngravingItem
     Millimetre userLength() const { return m_userLength; }
     void setUserLength(Millimetre userLength) { m_userLength = userLength; }
 
-    Millimetre lineWidth() const { return m_lineWidth; }
-    double lineWidthMag() const { return m_lineWidth * mag(); }
-    void setLineWidth(Millimetre lineWidth) { m_lineWidth = lineWidth; }
+    double point(Spatium sp) const override { return sp.val() * score()->style().spatium(); }
```

Comment:
I don't think this should use `score()->style().spatium()` though, or should it? Cause that returns the score-wide value, but this item may be on a small (or otherwise scaled) staff. We should always use `item->spatium()`.

Also, like Casper, I too prefer these implementations to be in the .cpp file (the idea being that .h files should #include as few other .h files as possible)

Reply:
This gives correct results - scaling is done in `lineWidthMag()` which scales by `mag()`.  `mag()` applies both the stave spatium and the chord's scale.  But actually maybe there's a UI issue I missed:
If you have a scaled stave, the "Thickness" property appears to the user to be measured in relation to the stave - (eg. by default always 0.1sp which is what we want).  If you then make the chord cue size, the stem's thickness is reduced, but the value appears to remain at 0.1sp.  
I can change `point` to use `item->spatium` and `lineWidthMag` to use `chord()->instrinsicMag` to make this more clear.  There might have to be some file compatibility changes for reading old files.